webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+
+
+module.exports = (env, argv) => {
+  const config = {
+    entry: {'react-streamfield': ['./src/index.js', './src/index.scss']},
+    output: {
+      path: path.resolve('dist'),
+      filename: '[name].js',
+      publicPath: '/static/'
+    },
+    module: {
+      rules: [
+        {
+          test: /\.jsx?$/,
+          loader: 'babel-loader',
+          exclude: /node_modules/,
+        },
+        {
+          test: /\.scss$/,
+          use: [
+            MiniCssExtractPlugin.loader,
+            'css-loader',
+            'sass-loader',
+          ]
+        },
+      ],
+    },
+    plugins: [
+      new MiniCssExtractPlugin({
+        filename: '[name].css',
+        chunkFilename: '[id].css',
+      }),
+    ],
+  };
+
+  if (argv.mode === 'production') {
+    config.optimization = {
+      minimizer: [
+        new UglifyJsPlugin({
+          cache: true,
+          parallel: true,
+          uglifyOptions: {
+            output: {
+              comments: false,
+              beautify: false
+            },
+            compress: {
+              drop_console: true,
```

Comment:
I'm not entirely sure what this does but it looks like a potential source of problems, as `console.warn` / `console.error` calls can be very useful in error handling. Their usage / presence should be enforced with linting instead.